substratevm/src/com.oracle.svm.hosted.jdk11/src/com/oracle/svm/hosted/ModuleLayerFeature.java

```diff
@@ -157,14 +188,79 @@ private ModuleLayer synthesizeRuntimeBootLayer(ImageClassLoader cl, Set<String>
         Configuration cf = synthesizeRuntimeBootLayerConfiguration(cl.modulepath(), reachableModules);
         try {
             ModuleLayer runtimeBootLayer = moduleLayerConstructor.newInstance(cf, List.of(), null);
-            Map<String, Module> nameToModule = nameToModuleSynthesizer.synthesizeNameToModule(runtimeBootLayer, cl.getClassLoader());
+            Map<String, Module> nameToModule = moduleLayerFeatureUtils.synthesizeNameToModule(runtimeBootLayer, cl.getClassLoader());
             patchRuntimeBootLayer(runtimeBootLayer, nameToModule);
             return runtimeBootLayer;
         } catch (InstantiationException | IllegalAccessException | InvocationTargetException ex) {
             throw VMError.shouldNotReachHere("Failed to synthesize the runtime boot module layer.", ex);
         }
     }
 
+    private void replicateVisibilityModifications(ModuleLayer runtimeBootLayer, ImageClassLoader cl, Set<Module> analysisReachableModules) {
+        List<Module> applicationModules = findApplicationModules(runtimeBootLayer, cl.applicationModulePath());
+
+        Map<String, HostedRuntimeModulePair> moduleLookupMap = analysisReachableModules
+                        .stream()
+                        .collect(Collectors.toMap(Module::getName, m -> new HostedRuntimeModulePair(m, runtimeBootLayer)));
+
+        Module builderModule = moduleLookupMap.get("org.graalvm.nativeimage.builder").hostedModule;
```

Comment:
Use `ModuleLayerFeature.class.getModule().getName()` instead of hardcoding `"org.graalvm.nativeimage.builder"`

Reply:
Or just `ModuleLayerFeature.class.getModule()` instead of `moduleLookupMap.get("org.graalvm.nativeimage.builder").hostedModule`